org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Sig.java

```diff
@@ -34,24 +34,27 @@
 import edu.mit.csail.sdg.alloy4.Version;
 import edu.mit.csail.sdg.ast.Attr.AttrType;
 
-/** Mutable; represents a signature. */
-
+/**
+ * Mutable; represents a signature.
+ *
+ * @modified Eduardo Pessoa, Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
supports the creation of variable/mutable sigs and fields; constructors are adapted accordingly, as are the creation of `Decl` objects (in this case only those of tricky fields are ever variable, the ones arising from parsed field declarations)